scripts/close_old_issues.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+import requests
+import json
+
+from datetime import datetime
+
+def parse_arguments():
+    """ parse command line arguments """
+    parser = argparse.ArgumentParser(description="Auto close issues")
+    parser.add_argument(
+        "-token",
+        required=True,
+        type=str,
+        help="Github access token",
+    )
+    return parser.parse_args()
+
+def get_issues(token: str):
+    params = {"Accept": "application/vnd.github+json",
+              "Authorization": f"token {token}",
+              "X-GitHub-Api-Version": "2022-11-28"}
+    url = f"https://api.github.com/repos/ewlu/gcc-precommit-ci/issues?per_page=100&state=open"
+    r = requests.get(url, params)
+    issues = json.loads(r.text)
+    filtered = [issue for issue in issues if 'pull_request' not in issue.keys()]
+    return filtered
+
+def get_patch_id(issue):
+    body = issue["body"].split("\n")
+    for line in body():
+        if line == "":
+            continue
+        if "Patch id:" in line:
+            patch_id = line.split(":")[-1].strip()
+            return patch_id
+    return None
+
+def get_patch_state(patch_id: str):
+    url = f"https://patchwork.sourceware.org/api/1.3/patches/{patch_id}"
+    r = requests.get(url)
+    details = json.loads(r.text)
+    return details["state"]
+
+def check_issue_is_closable(issue):
+    patch_id = get_patch_id(issue)
+    if patch_id is None:
+        return False
+    patch_state = get_patch_state(patch_id)
+    if patch_state != "committed" and patch_state != "superceded":
```

Comment:
Whoops, I double checked the spelling and got it wrong the first time:
`Superseded`